knowledge-base/error-handling-in-reports-and-report-viewers.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Error handling in Reports and Report Viewers
+description: Error handling in Reports and Report Viewers.
```

Comment:
As discussed, change the description - https://wiki.progress.com/display/DTS/Customer+Context